src/libratbag-private.h

```diff
@@ -215,6 +224,16 @@ struct ratbag_resolution{
 	uint32_t capabilities;
 };
 
+struct ratbag_led {
+	struct ratbag_profile *profile;
+	enum ratbag_led_type type;
+	enum ratbag_led_mode mode;  /**< off; on; cycle; breath  */
+	struct ratbag_color color; /**< led color  */
+	unsigned int rate;	/**< rate of action 100 - 20000 */
+	unsigned int bright;	/**< brightness of the LED 1 - 100 */
```

Comment:
0 isn't allowed?

I'm in two minds here: 0..255 would be a more machine natural range and matches some of the LED capabilities closer. 0..100% is a more human-natural range but has lower granularity. and if we use 0..100, it might be worth adding a typedef int percent_t to make the code more obvious about data ranges.